### PR TITLE
Update note

### PR DESCRIPTION
**Pull request by tapaswenipathak**
Fix https://github.com/readthedocs/readthedocs.org/issues/5908.

Add a line as the last point.

### CODE REVIEW

File: docs/development/install.rst [hunk 1]
```diff
@@ -35,6 +35,8 @@ Additionally Read the Docs depends on:
 
     If you plan to import Python 2 projects to your RTD,
     then you'll need to install Python 2 with virtualenv in your system as well.
+    The project works will all versions of Python 3.x, make sure you setup the
+    environment based on Python version you will be running in RTD.
```
Comment (stsewd) [inline docs/development/install.rst]: I think we can just rephrase the above sentence like: If you plan to import a project with any other python version...

Comment (tapaswenipathak) [reply docs/development/install.rst]: Does RTD works with Python 1.x series?

Comment (stsewd) [reply docs/development/install.rst]: 2.x and 3.x only, those are the supported versions from python.org

Comment (tapaswenipathak) [reply docs/development/install.rst]: Does RTD works with Python 1.x version?

Comment (tapaswenipathak) [reply docs/development/install.rst]: Sorry tethering issues, my comment now appears after yours. Should I rephrase saying

> `any other python version`

Comment (tapaswenipathak) [reply docs/development/install.rst]: I don't think we should say `any other python version` that's why I didn't. Lemme know 98f65c37728fd7ab7b02dba0ac5568e7faa493fc.

File: docs/development/install.rst [hunk 2]
```diff
@@ -33,8 +33,8 @@ Additionally Read the Docs depends on:
 
 .. note::
 
-    If you plan to import Python 2 projects to your RTD,
-    then you'll need to install Python 2 with virtualenv in your system as well.
+    You can import Python 2.x or 3.x projects in RTD, make sure you install the
+    appropriate Python version (2.xxx or 3.xxx) with virtualenv.
```
Comment (stsewd) [inline docs/development/install.rst]: Probably we can just put one `x` -`2.x or 3.x`

**Pull request merged**